util/odin/odin.go

```diff
@@ -148,15 +148,24 @@ func doRequest(method, path string, params url.Values, requestBody, responseBody
 	}
 
 	req.Header.Set("Content-Type", "application/json")
-	lumber.Debug("req: %+v\n", req)
+	lumber.Trace("REQ: %s %s %s", req.Method, req.URL, req.Proto)
```

Comment:
will have to make this `.Debug` in order to print with `-v`, but response contains the same infos